packages/cloudflare/src/index.ts

```diff
@@ -265,7 +265,26 @@ export default function createIntegration(args?: Options): AstroIntegration {
 								'ignored-bare-import': 'silent',
 							},
 							plugins: !args?.wasmModuleImports
-								? []
+								? [
+										{
+											name: 'sharp',
+											setup(build) {
+												if (args?.imageService === 'compile') {
+													build.onResolve({ filter: /^sharp/ }, (args) => ({
+														path: args.path,
+														namespace: 'sharp-ns',
+													}));
+
+													build.onLoad({ filter: /.*/, namespace: 'sharp-ns' }, (a) => {
+														return {
+															contents: JSON.stringify(''),
+															loader: 'json',
+														};
+													});
+												}
+											},
+										},
```

Comment:
Looks like this plugin is duplicated in this file. Would be nice to abstract it out as a function.

Reply:
Good point, I'll abstract it :)